x-pack/test/observability_functional/apps/observability/pages/alerts/custom_threshold_preview_chart.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from 'expect';
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+
+export default ({ getService, getPageObject }: FtrProviderContext) => {
+  const common = getPageObject('common');
+  const esArchiver = getService('esArchiver');
+  const testSubjects = getService('testSubjects');
+  const kibanaServer = getService('kibanaServer');
+  const supertest = getService('supertest');
+  const find = getService('find');
+  const logger = getService('log');
+  const retry = getService('retry');
+
+  describe('Custom threshold preview chart', () => {
+    const observability = getService('observability');
+    const DATA_VIEW_1 = 'metricbeat-*';
+    const DATA_VIEW_1_ID = 'data-view-id_1';
+    const DATA_VIEW_1_NAME = 'test-data-view-name_1';
+
+    before(async () => {
+      await esArchiver.load('x-pack/test/functional/es_archives/infra/metrics_and_logs');
+      await observability.alerts.common.createDataView({
+        supertest,
+        name: DATA_VIEW_1_NAME,
+        id: DATA_VIEW_1_ID,
+        title: DATA_VIEW_1,
+        logger,
+      });
+      await observability.alerts.common.navigateToRulesPage();
+      // TODO Remove when it's fixed https://github.com/elastic/kibana/issues/201805
+      await common.sleep(1000);
+      if (await testSubjects.exists('toastCloseButton')) {
+        await testSubjects.click('toastCloseButton');
```

Comment:
When testing locally, removing this part didn't affect the test, shall we remove this logic?